### ISSUE TITLE
Add info on qq quotes for behavior of left angle brackets

### ISSUE THREAD

**Issue by tbrowder** [Raku/doc#3940]
## Problem or new feature

Given the following code:

    my $a = 'foo';
    say qq:to/HERE/;
        <a>$a</a>
    HERE
    
one gets this result:

    (to be entered shortly)

The docs should expand a bit more on such behavior and its mitigation.

**Comment by jonathanstowe** [comment]
I get

```
Type Str does not support associative indexing.
  in block <unit> at -e line 3
```

That can be fixed by doing either:

```
my $a = "foo";
say qq:to/HERE/;
    <a>{$a}</a>
HERE
```

or

```
my $a = "foo";
say qq:to/HERE/;
    <a>$a\</a>
HERE
```

It's not so much the left angle brackets, rather the constructs that look like indexing.

**Comment by jonathanstowe** [comment]
You'd get similar if the characters after the variable were `{..}` or `[..]` for instance.

**Comment by tbrowder** [comment]
You are correct, and that's how I fixed my problem (with {$a}, my goto solution for qq problems). What was confusing me in my case was where the problem was since the code in the HERE block is quite lengthy with javascript, html, and other embedded strings I had already put in curlies (as well as escaped curlies in the js code). 

I finally found the offending line by starting with the first line only and gradually adding one line at a time. I had failed to spot the problem because it was so innocent looking!

**Comment by tbrowder** [comment]
I will offer a PR later.

**Comment by tbrowder** [comment]
See PR #3941
